src/config/config.h

```diff
@@ -127,7 +127,7 @@ struct Config {
   std::string master_host;
   std::string unixsocket;
   int unixsocketperm = 0777;
-  int master_port = 0;
+  uint32_t master_port = 0;
```

Comment:
After this change config has the variable `port_` of int type and `master_port_` of uint32_t which is some kind of inconsistent. I also noticed that somewhere across the codebase we cast `uint32_t` to `int` with a port value, but my thought at that moment was to leave the variable of type `int` and change `uint32_t` to `int` in function signatures through the call chains.
However, I see that for example, the `ntohs` function returns an unsigned value.
@PragmaTwice What do you think?

Reply:
I think port number should be a `uint16_t`, but `uint32_t` is fine to me.

Reply:
Changed `port` and `tls_port` types from `int` to `uint32_t`